### PR TITLE
List all resources in a succinct way and remove TPB

### PR DESCRIPTION
**Pull request by radixhound**

### CODE REVIEW

File: hubot-scripts.json [hunk 1]
```diff
@@ -1 +1 @@
-["redis-brain.coffee", "tweet.coffee", "shipit.coffee", "reddit-jokes.coffee", "tpb.coffee", "xkcd.coffee"]
+["redis-brain.coffee", "tweet.coffee", "shipit.coffee", "reddit-jokes.coffee", "xkcd.coffee"]
```
Comment (tjwallace) [inline hubot-scripts.json]: -1

Comment (radixhound) [inline hubot-scripts.json]: If you want to swear then do it yourself don't get a robot to do it for you.

Comment (tjwallace) [inline hubot-scripts.json]: I don't want to swear, I want funny Trailer Park Boys quotes

Comment (radixhound) [inline hubot-scripts.json]: It's TBP jokes laden with extremely crude language, which are only funny if you watch the show. If you're not willing to say this out loud yourself in front of _everyone_ in the company, then don't get a robot to say it in our chat rooms. There are even potential legal implications, so let's not eh?

Comment (tjwallace) [inline hubot-scripts.json]: What are the potential legal implications (I'm genuinely curious)?

Comment (radixhound) [inline hubot-scripts.json]: Crude sexual jokes can be considered a form of harassment. As we grow our team we should be sensitive to that. Not everyone will express their discomfort openly.

Comment (evanhourigan) [inline hubot-scripts.json]: I understand the objection to jokes that could be considered offensive, though to be fair, these are quotes picked at random, and not said in context to anything.  As a big TPB fan, `-1`.  I'm okay with forking a modified version of the original script.

**Pull request merged**